tests/test_strtree.py

```diff
@@ -1,214 +0,0 @@
-import gc
```

Comment:
All tests are either superseded by more comprehensive test cases in `shapely/tests/test_strtree.py` or were migrated there (pickle, nearest exclusive).  Tests for custom items / geometries would be implemented if / when we implement specific classes for such (as per [#1386 comment](https://github.com/shapely/shapely/issues/1386#issuecomment-1147765811))